install_sct

```diff
@@ -527,6 +527,11 @@ yes | python/bin/conda create -n venv_sct python=3.6
 # activate miniconda
 source python/etc/profile.d/conda.sh
 conda activate venv_sct
+if [ "$(which python)" != "$(which python/envs/venv_sct/bin/python)" ]; then
```

Comment:
Alternatively, a different check could be:

```suggestion
if [ "$(echo $PATH | cut -f1 -d:)" != "$SCT_DIR/python/envs/venv_sct/bin" ]; then
```

Which covers all the related binaries.